petastorm/reader_impl/row_group_decoder.py

```diff
@@ -72,3 +73,40 @@ def decode(self, row):
             return row
         else:
             return self._schema.make_namedtuple(**utils.decode_row(row, self._schema))
+
+    def debundle_row(self, row):
```

Comment:
Should we call the other implementation from here? Maybe should think about consolidating the bundling/debundling functionality into a standalone class/module as mentioned in another comment.